### DISCUSSION TITLE
Disable 'ha banner'

### DISCUSSION THREAD

**Discussion by mikael-andersson** [hassio-addons/addon-ssh#375] [Ideas]
When I invoke an SSH command, the 'ha banner' defined in /etc/profile.d/homeassistant.sh is displayed, 
Could we add an option for enabling/disabling it in https://github.com/hassio-addons/addon-ssh/blob/main/ssh/rootfs/etc/cont-init.d/user.sh? If it sounds OK, what is the best way of doing this? When we have a suggestion, I can probably create a pull request.

**Comment by frenck** [comment]
I have no intention to add options for small cosmetic changes like these. You can use the init command configuration option to e.g., run a sed command or similar to change behaviors to your likings.

**Reply by mikael-andersson** [reply]
Thanks for the prompt answer! :) Then I will do that, thanks.

**Reply by mikael-andersson** [reply]
What am I doing wrong? The below simple test init_commands does not create a file as I expect. Yesterday I also noticed that I am unable to create a user other than the default root.

apks: []
authorized_keys:
  - >-
    ssh-rsa
    AAAAB3NzaC....
password: ''
server:
  tcp_forwarding: false
username: testuser
init_commands:
  - touch /config/hejhopp

**Reply by frenck** [reply]
Are you mixing up add-ons? `apks` sounds like the core SSH add-on, not the community version

**Reply by mikael-andersson** [reply]
Oops, it certainly sounds like I am mixing things up. I was not aware that there were two SSH add-on versions..., but I can see it now. :)

Yes, I am using the core version.

It sounds like I should change add-on then. :) What are the advantages with the core version, stability and security?

**Reply by frenck** [reply]
Mainly extensiveness. The community version is more aimed at being a swiss army knife.

**Reply by mikael-andersson** [reply]
Thanks for the answer. What is the disadvantage with only one version, i.e. the community version, which is more capable?

**Reply by frenck** [reply]
With great power comes great responsibility...
That is the main thing, the community version gains more privileges from the system and does could do more harm if used incorrectly/inresonsibly.

**Reply by mikael-andersson** [reply]
Sounds reasonable. :) I will definitely have a look at the community version, thank you very much for your time and for all your great work! I am one of your sponsors now. :)

**Reply by frenck** [reply]
Thank you! That is appreciated ❤️
